### PR TITLE
Prevent title from appearing when opening sharing tooltip

### PR DESCRIPTION
**Pull request by karimJWP**
### This PR will...
hide title only on small player.

### Why is this Pull Request needed?
Title should NOT appear when sharing tooltip is opened.

### Are there any points in the code the reviewer needs to double check?
n/a

### Are there any Pull Requests open in other repos which need to be merged with this?
n/a

#### Addresses Issue(s):
JW7-4415

### CODE REVIEW

File: src/css/controls/flags/overlay.less [hunk 1]
```diff
@@ -1,13 +1,9 @@
...
-        display: none;
-    }
-
     .jw-controls-right .jw-logo {
         display: none;
     }
 
-    &-sharing,
+    &-sharing.jw-flag-small-player,
     &-related {
```
Comment (hghazzi) [inline src/css/controls/flags/overlay.less]: Should this happen for the Playlist/Discovery tooltip

Comment (karimJWP) [inline src/css/controls/flags/overlay.less]: @hghazzi yup, related is a full overlay, so title and controls disappear

**Comment by karimJWP** [comment]
retest this

File: src/css/controls/flags/overlay.less [hunk 2]
```diff
@@ -1,13 +1,9 @@
 .jwplayer.jw-flag-overlay-open {
-    .jw-title {
```
Comment (egreaves) [inline src/css/controls/flags/overlay.less]: Let's keep this rule for all overlays and show the title for the exception.

File: src/css/controls/flags/overlay.less [hunk 3]
```diff
@@ -17,7 +13,7 @@
 
     &-sharing:not(.jw-flag-small-player) {
         .jw-controls,
-        .jw-title {
+        .jw-controls-right .jw-logo {
```
Comment (egreaves) [inline src/css/controls/flags/overlay.less]: This is the exception where the title should be shown, so let's keep `.jw-title` here

**Comment by karimJWP** [comment]
@egreaves changes addressed

**Pull request merged**